release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -58,14 +58,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-installation-and-update"]
 === Installation and update
 
-The Cluster API Provider for OpenStack (CAPO) is now deployed by default when the `TechPreviewNoUpgrade` feature flag is enabled. The lifecycle of this component is managed by the CAPI Operator. When the `TechPreviewNoUpgrade` feature flag is enabled, an OpenStack cluster and additional OpenShift cluster are automatically created for the current OpenShift cluster. It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that these resources are equivalent but not identical.
-
-[id="ocp-4-15-installation-and-update-ibm-cloud-user-managed-encryption"]
-==== IBM Cloud and user-managed encryption
-
-You can now specify your own {ibm-name} Key Protect for {ibm-cloud-name} or {ibm-name} Hyper Protect Crypto Services root key as part of the installation process. This root key is used to encrypt the root (boot) volume of control plane and compute machines, as well as the persistent volumes (data volumes) that are provisioned after the cluster is deployed.
+[discrete]
+=== CAPO integration into the cluster CAPI Operator (Tech Preview)
 
-For more information, see xref:../installing/installing_ibm_cloud_public/user-managed-encryption-ibm-cloud.adoc#user-managed-encryption-ibm-cloud[User-managed encryption for IBM Cloud].
+The Cluster API Provider for OpenStack (CAPO) is now deployed by default when the `TechPreviewNoUpgrade` feature flag is enabled. The lifecycle of this component is managed by the Coherent Accelerator Processor Interface (CAPI) Operator. When the `TechPreviewNoUpgrade` feature flag is enabled, an OpenStack cluster and additional {product-title} cluster are automatically created for the current {product-title} cluster. It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that these resources are equivalent but not identical.
```

Comment:
There is a lot of passive voice here. In my view this should use the active voice. Something like:

If you enable the `TechPreviewNoUpgrade` feature flag, the Coherent Accelerator Processor Interface (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.

The part about configuring the CAPI machine should be a separate paragraph and also use the active voice: "You can now configure..."

The "equivalent but not identical" sentence needs to be expanded in my opinion as right now it is very hard to guess the meaning (I could not). It should probably be a separate paragraph too.

Reply:
Thank you for the feedback - I've adjusted :) the last sentence relates to the resources of CAPI vs MAPI. I've rephrased a bit.